Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/8PI6-pool-network-delay/pool-network-delay

```diff
@@ -74,7 +128,61 @@ python3 Openshift-EE/utils/result/result_update.py $job_id 8PI6 4-chaos "Induce
 exit 1;
 fi
 
-echo "********Deprovisioning Busybox Application*******"
+################
+# LitmusBook 4 #
+################
+
+run_id="deprovision-pool-network-delay";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-liveness metadata=${run_id})
+echo $test_name
+
+cd litmus
+# copy the content of deployer run_litmus_test.yml into a different file to update the test specific parameters.
+cp apps/busybox/liveness/run_litmus_test.yml busybox_loadgen_deprovision_pool_network_delay.yml
+
+# Update the environmental variables in litmus job spec.
+
+: << EOF
+  ---------------------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                                          |
+  --------------------------------------------------------------------------------------------------------------------------------|
+ | appNamespace      | env    | app-busybox-ns                      | pool-delay                                                  | 
+ | appLabel          | env    | app=busybox-sts                     | app=pool-delay                                              |
+ | Litmus job label  | label  | liveness: litmus-busybox-liveness   | liveness: busybox-liveness-deprovision-pool-network-delay   |
+ | action            | env    | provision                           | deprovision                                                 | 
+  --------------------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: app-busybox-ns/value: pool-delay/g' \
+-e 's/app=busybox-sts/app=pool-delay/g' \
+-e 's/value: provision/value: deprovision/g' \
+-e 's/liveness: litmus-busybox-liveness/liveness: busybox-liveness-deprovision-pool-network-delay/g' busybox_loadgen_deprovision_pool_network_delay.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' busybox_loadgen_deprovision_pool_network_delay.yml
+
+cat busybox_loadgen_deprovision_app_kill.yml
```

Comment:
@harshita-sharma011 check the file `cat busybox_loadgen_deprovision_app_kill.yml`